### DISCUSSION TITLE
Multithread/ multiprocessing capability

### DISCUSSION THREAD

**Discussion by buddhiwisr** [Evolutionary-Intelligence/pypop#180] [Q&A] [answered]
Is it possible to incorporate multithread/ multiprocessing capability in PSO calculations?

**Comment by Evolutionary-Intelligence** [comment, marked as answer]
We are planning to add this capability into this new open-source library https://github.com/Evolutionary-Intelligence/dpop7

**Comment by Evolutionary-Intelligence** [comment]
@buddhiwisr If possible, welcome to make open-source contributions for this new open-source library **dpop7**
